pkg/apb/svc_acct.go

```diff
@@ -0,0 +1,197 @@
+package apb
+
+import (
+	logging "github.com/op/go-logging"
+	yaml "gopkg.in/yaml.v2"
+	"os"
+	"path/filepath"
+)
+
+type ServiceAccountManager struct {
+	log *logging.Logger
+}
+
+func NewServiceAccountManager(log *logging.Logger) ServiceAccountManager {
+	return ServiceAccountManager{
+		log: log,
+	}
+}
+
+// Sets up ServiceAccount based apb sandbox
+// Returns service account name to be used as a handle for destroying
+// the sandbox at the conclusion of running the apb
+func (s *ServiceAccountManager) CreateApbSandbox(
+	namespace string,
+	apbId string,
+) (string, error) {
+	svcAccountName := apbId
+	roleBindingName := apbId
+	apbRole := "cluster-admin"
```

Comment:
I'm not so worried about it being a const, but it's exposure.  Normally,  I would think a variable like this would end up somewhere in types because it's the default Role value for the service account.